superset/security.py

```diff
@@ -727,6 +773,7 @@ def _is_granter_pvm(self, pvm: PermissionModelView) -> bool:
 
         return pvm.permission.name in {"can_override_role_permissions", "can_approve"}
 
+    # TODO(bogdan): add unit tests
```

Comment:
It would be great if this TODO was addressed as part of this PR, especially given that this file is named `security.py`

Reply:
actually I did, forgot to remove the TODO. 
See test_set_perm_<data_model> tests in the security_tests.py

will remove todo